src/lib/components/Modal/Modal.jsx

```diff
@@ -184,6 +253,10 @@ Modal.propTypes = {
     // eslint-disable-next-line react/forbid-prop-types
     current: PropTypes.any,
   }),
+  /**
+   * Enforces focus to stay inside of Modal. Works only if autoFocus is not disabled.
```

Comment:
I wanted to express the fact that `autoFocus` is enabled by default and it needs to be explicitly disabled, which then also disables focus trap making no sense provided the focus is not moved into the `Modal`.

Reply:
From my point of view, it's the same story as with `if (!disableFocusTrap)`, just in words instead of code. The default value should have no impact on the API itself.

I'm not going to push here, but I'd prefer to have it more straightforward, which means easier to understand and easier to maintan.